src/app/features/course/course.component.html

```diff
@@ -0,0 +1,7 @@
+<info class="text">{{text}}</info>
+<info class="title">{{title}}</info>
+<ul class='information-list'>
+    <li><b>Authors: </b>{{authors}}</li>
+    <li class="duration"><b>Duration: </b>{{duration}}</li>
```

Comment:
![image](https://user-images.githubusercontent.com/33813087/151661724-49e44e17-8c63-43d3-b86b-84f0b25e16b6.png)

Reply:
resolved in task N3